packages/dev/docs/src/PropTable.js

```diff
@@ -81,14 +81,14 @@ export function PropTable({component, links, style}) {
   return (
     <div style={style}>
       <TypeContext.Provider value={links}>
-        <InterfaceType properties={ungrouped} showRequired showDefault isComponent name={component.name} />
+        <InterfaceType properties={ungrouped} showRequired isComponent name={component.name} />
```

Comment:
Props tables now only show the "default" column if any of the props actually have a default.